altair/vegalite/v5/schema/_config.py

```diff
@@ -0,0 +1,1055 @@
+    startAngle: float
+    stroke: str | None | LinearGradient | RadialGradient | ColorName_T
+    strokeCap: StrokeCap_T
+    strokeDash: Sequence[float]
+    strokeDashOffset: float
+    strokeJoin: StrokeJoin_T
+    strokeMiterLimit: float
+    strokeOffset: float
+    strokeOpacity: float
+    strokeWidth: float
+    tension: float
+    text: str | Sequence[str]
+    theta: float
+    theta2: float
+    timeUnitBandPosition: float
+    timeUnitBandSize: float
+    tooltip: str | bool | None | float | TooltipContent
+    url: str
+    width: float
+    x: str | float
+    x2: str | float
+    y: str | float
+    y2: str | float
+
+
+class AreaConfigKwds(TypedDict, total=False):
+    """Placeholder doc."""
+
+    align: Align_T
+    angle: float
+    aria: bool
+    ariaRole: str
+    ariaRoleDescription: str
+    aspect: bool
+    baseline: str | Baseline_T
+    blend: Blend_T
+    color: str | LinearGradient | RadialGradient | ColorName_T
+    cornerRadius: float
+    cornerRadiusBottomLeft: float
+    cornerRadiusBottomRight: float
+    cornerRadiusTopLeft: float
+    cornerRadiusTopRight: float
+    cursor: Cursor_T
+    description: str
+    dir: TextDirection_T
+    dx: float
+    dy: float
+    ellipsis: str
+    endAngle: float
+    fill: str | None | LinearGradient | RadialGradient | ColorName_T
+    fillOpacity: float
+    filled: bool
+    font: str
+    fontSize: float
+    fontStyle: str | FontStyle
+    fontWeight: FontWeight_T
+    height: float
+    href: str
+    innerRadius: float
+    interpolate: Interpolate_T
+    invalid: None | MarkInvalidDataMode_T
+    limit: float
+    line: bool | OverlayMarkDef
+    lineBreak: str
+    lineHeight: float
+    opacity: float
+    order: bool | None
+    orient: Orientation_T
+    outerRadius: float
+    padAngle: float
+    point: str | bool | OverlayMarkDef
+    radius: float
+    radius2: float
+    shape: str | SymbolShape
+    size: float
+    smooth: bool
+    startAngle: float
+    stroke: str | None | LinearGradient | RadialGradient | ColorName_T
+    strokeCap: StrokeCap_T
+    strokeDash: Sequence[float]
+    strokeDashOffset: float
+    strokeJoin: StrokeJoin_T
+    strokeMiterLimit: float
+    strokeOffset: float
+    strokeOpacity: float
+    strokeWidth: float
+    tension: float
+    text: str | Sequence[str]
+    theta: float
+    theta2: float
+    timeUnitBandPosition: float
+    timeUnitBandSize: float
+    tooltip: str | bool | None | float | TooltipContent
+    url: str
+    width: float
+    x: str | float
+    x2: str | float
+    y: str | float
+    y2: str | float
+
+
+class AxisConfigKwds(TypedDict, total=False):
+    """Placeholder doc."""
+
+    aria: bool
+    bandPosition: float
+    description: str
+    disable: bool
+    domain: bool
+    domainCap: StrokeCap_T
+    domainColor: str | None | ColorName_T
+    domainDash: Sequence[float]
+    domainDashOffset: float
+    domainOpacity: float
+    domainWidth: float
+    format: str | Dict
```

Comment:
TODO: `Dict`

This is `core.Dict`

Reply:
Both occurrences are for `format: str | Dict`.

- https://vega.github.io/vega-lite/docs/header.html#labels
- https://vega.github.io/vega-lite/docs/format.html

Reading through the `vega-lite` docs, I can't see how this would be used.

---

Related, didn't realise `Dict` was just an aliased js `Object`. 
I suppose we could use `Map`, but an example of how this is used would be helpful

Reply:
Fixed in [`b4f7ede` (#3536)](https://github.com/vega/altair/pull/3536/commits/b4f7ededff4c5f4f6371f6d0f049b4766e99c74b)